src/components/About/Header/index.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+import "../index.scss";
```

Comment:
Have components css file inside same folder

Reply:
accidentally put the file in the wrong place, my fault, the rest did so